addOns/reports/src/main/java/org/zaproxy/addon/reports/ReportData.java

```diff
@@ -130,4 +132,24 @@ public boolean isIncludeRisk(int risk) {
         }
         return false;
     }
+
+    public List<String> getSections() {
+        return sections;
+    }
+
+    public void addSection(String section) {
+        this.sections.add(section);
+    }
+
+    public void removeSection(String section) {
+        this.sections.remove(section);
+    }
+
+    public boolean isIncludeSection(String section) {
+        return this.sections != null && this.sections.contains(section);
+    }
+
+    public void setSections(List<String> sections) {
```

Comment:
`set` → `add`, or is the method intended to replace all sections? (in which case it should clear before adding all).